### ISSUE TITLE
`git machete go u` ends up in an `Invalid direction` message

### ISSUE THREAD

**Issue by PawelLipski** [VirtusLab/git-machete#226]
`go up` works as expected, tho. All other one-letter abbrevs seems to work correctly, too.

```
$ git machete g u

Invalid direction: u expected: d[own]|f[irst]|l[ast]|n[ext]|p[rev]|r[oot]|u[p]
```

**Also, the error message should have some comma/semicolon before `expected`.**

Within the scope of this task, a regression test should be added (for `u` and for other 1-letter shortcuts, both for `show` and `go`).

**Comment by rifel123** [comment]
Due to regression introduced in #212 :
<img width="874" alt="Screenshot 2021-09-20 at 10 21 08" src="https://user-images.githubusercontent.com/73847352/133974095-1ba5ab00-0751-4d17-a13a-99096924b99b.png">

**Comment by rifel123** [comment]
closed with #256
